src/wwwroot/template-editor/adapterService.js

```diff
@@ -354,17 +375,28 @@
      */
     function getCampaign(id, useEditorAsTemplate) {
       traceAdapterCall(getCampaign, arguments);
-      var campaign = {
-        id: id,
-        type: 'campaign',
-        name: 'Campaign ' + id,
-        attributes: {},
-        innerHTML: '',
-        children: []
-      };
-      return $q.resolve({ data: campaign });
+      return $http.get(`${RELAY_CONFIG.baseUrl}/accounts/${loginSession.accountName}/template/${id}`, {
```

Comment:
Hehe, it is my fault, yesterday I asked you to use _accountName_ in place of _accountId_, but all template methods for the moment are using _accountId_.

Anyway, we will call another URL, for example, something like `/accounts/{accountName}/template/{id}/content`, so, what do you think about to go back to _accontId_ and add a _TODO_ comment indicating that it is a temporal URL?

Reply:
It good for me, I will revert it.